src/App/Fossa/Analyze.hs

```diff
@@ -180,33 +181,38 @@ analyze basedir destination override unpackArchives filters = do
   case destination of
     OutputStdout -> logStdout $ pretty (decodeUtf8 (Aeson.encode (buildResult projectResults)))
     UploadScan apiOpts metadata -> do
-      revision <- mergeOverride override <$> inferProject (unBaseDir basedir)
+      let maybeResults = NE.nonEmpty projectResults
+      case maybeResults of
+        Nothing -> logError "No analysis to upload"
```

Comment:
I think it would be helpful if we added some more information here like "the fossa cli did not detect any dependencies in this directory", this message right now is a little unclear about whats happening and I don't think all uses will know what "No analysis" means.

Reply:
I agree with the point, but it's not "dependencies", it's really analyses.  In this state we couldn't find anything that we could even **attempt** to analyze.  If we analyzed, but found 0 dependencies, this message isn't shown.